### DISCUSSION TITLE
为什么看这么多“没用的”东西

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#140] [Announcements]
https://www.bbing.com.cn/202109/whyread-useless-content/

**Comment by ryan4yin** [comment]
特地前来读一读这篇「没用的书」，顺便说下我的想法吧。
每个想要精进的技术人，其实都免不了要去学习这些底层知识，单纯专注于上层也能有很大成就的人不能说没有，只能说是极少数。


我学习密码学、分布式系统与数据库的原因，一部分就是因为理解到这一点，另一部分原因是觉得自己还太弱了，想要去了解。
